auth.go

```diff
@@ -186,3 +223,11 @@ func readTemplate(file string) (*template.Template, error) {
 
 	return template.New("").Parse(string(bytes))
 }
+
+func getPath(r *http.Request) string {
+	return r.URL.Path
+}
+
+func isIgnoredUri(uri string) bool {
```

Comment:
i wouldn't mind checking this. In any case, if it's handled it should be configurable via the config options to specify a file and serve it